common/src/main/java/org/apache/hertzbeat/common/entity/job/protocol/HttpsdProtocol.java

```diff
@@ -34,10 +34,10 @@
 @NoArgsConstructor
 public class HttpsdProtocol {
     private String host;
-    private String port;
+    private int port;
```

Comment:
the port type in yml need string, due we need string param like `^_^port^_^`

Reply:
In `HttpsdProtocol.java`, I've changed `string` to `int` to be consistent with the port database type in the service discovery component.

Reply:
yes, but the param `port` is custom, user can custom the port value with `^_^port^_^`, if here we input the `^_^port^_^` and port type is int, will casue error.

Reply:
got it, I will restore it.